rest_framework_api_key/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.3"
+__version__ = "0.0.4"
```

Comment:
Instead of possibly hitting a conflict, I'd suggest `0.0.3.post1` per [PEP440](https://www.python.org/dev/peps/pep-0440/) maybe?